lambdas/db-migration/src/migrations/20201014105054_create_pdrs_table.ts

```diff
@@ -22,8 +19,36 @@ export const up = async (knex: Knex): Promise<void> =>
     table
       .integer('executionCumulusId')
       .references('cumulusId')
-      .inTable('executions')
+      .inTable('executions');
+    table
+      .enum('status', ['running', 'failed', 'completed'])
+      .comment('Status (running, failed, completed) of the PDR')
       .notNullable();
+    table
+      .float('progress')
+      .comment('PDR completion progress percentage');
+    table
+      .boolean('PANSent')
+      .comment('Boolean defining if a PAN response has been sent for this PDR');
+    table
+      .text('PANmessage')
```

Comment:
Given talking about changing the column names, perhaps it doesn't matter, but PANSent vs PANmessage is mapped to the existing dynamo columns.

Reply:
Do you mean that we should change the column names?

Reply:
@markdboyd  that was a reference to wanting to change things like originalUrl -> original_url to be more consistent with postgres expectations.

Reply:
Yeah, if we're going to change column names to snake_case regardless, then I'm fine with changing these too

Reply:
Right, I think just leaving them in *this* PR is fine for now though, as we're going to have a PR to wholesale change them.